### ISSUE TITLE
I got a problem of generating flipped images.

### ISSUE THREAD

**Issue by yufeilongyuan** [Belval/TextRecognitionDataGenerator#73]
Hello, Author. Thank you for your contribution to data generation. But after use, there is a question to consult you. The problem is: I know that the generated image can be vertical, but I want to flip 90 degrees to the left when the image is vertical, so that the font will flip. Excuse me, how to modify the code, I hope to get your opinion, thank you!

**Comment by Belval** [comment]
I am not 100% sure that I understand your question. Here is my best shot at making what you are asking.

Vertical + Rotation clockwise
`python3 run.py -c 1 -l cn -w 10 -sw 0 -or 1 -k 270 -f 200`

![皆 群 無 月 為 惡 裝 店 想 響_0](https://user-images.githubusercontent.com/5399488/60612861-b0012b80-9d97-11e9-80a2-241e9218e22b.jpg)

Vertical + Rotation counter clockwise
`python3 run.py -c 1 -l cn -w 10 -sw 0 -or 1 -k 90 -f 200`

![使 迎 付 鳥 配 毛 賞 嘛 欣 支_3](https://user-images.githubusercontent.com/5399488/60612896-c7401900-9d97-11e9-8da1-362df6939bf5.jpg)

In plain English, this command says "Make 1 example in Chinese with 10 characters with 0 space between the characters, vertical, and rotated 90 degrees counter clockwise."

Is that what you want?

**Comment by yufeilongyuan** [comment]
Thank you for taking the time to answer my questions. Thank you again for your answers. But I have a new problem, that is, when generating many images, such as 100, there will be no text, empty content, whether by default or using their own text. I don't know what caused the result. I hope you can see the news and take some time to give some suggestions. Thank you.

**Comment by yufeilongyuan** [comment]
I've found out what the problem is, and I hope to help more people. The reason is very simple. There are too few font libraries. When you can't choose them immediately, there will be blank content. Adding a few more font libraries can significantly improve, and I hope it will be helpful to you. Thank you again for the contribution of the author. I am working hard on the source code. Thank you.
